app/controllers/application_controller.rb

```diff
@@ -73,7 +72,17 @@ def analytics
   end
 
   def analytics_user
-    effective_user || AnonymousUser.new
+    hybrid_user || AnonymousUser.new
+  end
+
+  def hybrid_user
```

Comment:
can we update the naming to something more reflective that this can be a current user or a hybrid user?